vocabularies/HTML5.json

```diff
@@ -0,0 +1,59 @@
+{
+    "$Version": "4.0",
+    "$Reference": {
+        "https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.json": {
+            "$Include": [
+                {
+                    "$Namespace": "Org.OData.Core.V1",
+                    "$Alias": "Core"
+                }
+            ]
+        },
+        "https://sap.github.io/odata-vocabularies/vocabularies/Common.xml": {
+            "$Include": [
+                {
+                    "$Namespace": "com.sap.vocabularies.Common.v1",
+                    "$Alias": "Common"
+                }
+            ]
+        }
+    },
+    "com.sap.vocabularies.HTML5.v1": {
+        "$Alias": "HTML5",
+        "@Common.Experimental": true,
+        "@Core.Description": "Terms for HTML5",
+        "@Core.LongDescription": "The HTML5 Vocabulary provides rendering hints for HTML5 clients",
+        "@Core.Description#Published": "2020-04-21 © Copyright 2020 SAP SE. All rights reserved",
+        "@Core.Links": [
+            {
+                "rel": "alternate",
+                "href": "https://sap.github.io/odata-vocabularies/vocabularies/HTML5.xml"
+            },
+            {
+                "rel": "latest-version",
+                "href": "https://sap.github.io/odata-vocabularies/vocabularies/HTML5.json"
+            },
+            {
+                "rel": "describedby",
+                "href": "https://github.com/sap/odata-vocabularies/blob/master/vocabularies/HTML5.md"
+            }
+        ],
+        "CssDefaults": {
+            "$Kind": "Term",
+            "$Type": "HTML5.CssDefaultsType",
+            "$AppliesTo": [
+                "Record"
+            ],
+            "@Common.Experimental": true,
+            "@Core.Description": "CSS definitions for e.g. UI.DataFieldAbstract annotations"
+        },
+        "CssDefaultsType": {
+            "$Kind": "ComplexType",
+            "@Common.Experimental": true,
+            "width": {
+                "$Nullable": true,
+                "@Core.Description": "css: width"
```

Comment:
Shall we refere somehow to the spec? https://www.w3.org/TR/CSS21/visudet.html#propdef-width

And I would add to the description something like.

`The property allows all values specified for the  original css width property. Note that any client consuming this annotation may only support selected length units`

Reply:
Added the reference and description